pom.xml

```diff
@@ -50,9 +51,75 @@
       <version>${version.assertj}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>org.jetbrains.kotlin</groupId>
+      <artifactId>kotlin-stdlib-jdk8</artifactId>
+      <version>${version.kotlin}</version>
+    </dependency>
```

Comment:
Esta dependencia debería ser de tipo provided  no? Si no como interactua con los proyectos no-kotlin que usan java-spec?
http://andresalmiray.com/maven-scopes-vs-gradle-configurations/

Reply:
Sí. Otra alternativa para investigar: https://maven.apache.org/guides/introduction/introduction-to-optional-and-excludes-dependencies.html